File: sumo-cuda.py
#!/usr/bin/python
##
# @file: car.py
# @author: Chris Blatchley
# @author: Thad Bond
# @date: Sun, Jan 5, 2014
# @version: 0.1
# ##
# Car object implementation.
##

from car import Car

help_string = """SUMO-CUDA
    USAGE: python sumo-cuda.py [options] network.netccfg

Authors: Thaddeus Bond, Chris Blatchley
"""

def main():
    miata = Car(0, 60, 5, 2)
    miata.honk()

if __name__ == '__main__':
    main()

File: src/sumo_cuda.cpp
/**
 * @file: sumo_cuda.cpp
 * @author: Chris Blatchley
 * @author: Thad Bond
 *
 * Main entry point for sumo-cuda
 */

#include <string>
#include <fstream>
#include <iostream>

void printHelpString()
{
    printf("SUMO-CUDA\n");
    printf("    USAGE: sumo-cuda [options] network.netccfg\n");
    printf("\n");
    printf("Authors: Thaddeus Bond, Chris Blatchley\n");
}

void test()
{
}

int int main(int argc, char const *argv[])
{
    if (argc > 1)
    {
        if ( argv[1] == "test" )
        {
            test();
            return 0;
        }
    }
    printHelpString();
    return 0;
}
